api/_examples/http-server-api.ts

```diff
@@ -15,7 +15,7 @@ import * as http from "http";
 import micro from "micro";
 import FanoutGraphqlApolloConfig, {
   FanoutGraphqlGripChannelsForSubscription,
-} from "../FanoutGraphqlApolloConfig";
+} from "../_lib/FanoutGraphqlApolloConfig";
```

Comment:
I've never seen underscore-prefixed directory names in any project. If I did encounter them, I'd assume they were temporary dirs or otherwise private/hidden. Why do this?

Reply:
I can move this to a `/lib` directory in the root if you prefer, but then I would need to change the tsconfig to include compiling that directory too.

The dirs/files prefixed with underscore will not be used as entrypoints to Lambdas. Since we are using zero config, the file/folder structure is important.

https://zeit.co/blog/zero-config

Reply:
> moved /src to /api

>> Why? Is this required to use now?

It is not required but I was under the impression that I could rename files folders so I opted for zero config instead of creating builds/routes in `now.json`.

Reply:
ok.

This is why I've been meh on now. Having to rename folders is not 'zero config', it's just moving config to convention of filenames instead of a config file. :)

Thank for pointing me to the relevant docs on how to configure the files to work with `now`, :P